concordia/converters.py

```diff
@@ -5,3 +5,8 @@ class UnicodeSlugConverter(SlugConverter):
     # This is similar to the slug_unicode_re pattern but is not anchored to the
     # start of the string:
     regex = r"[-\w+]+"
+
+
+class ItemSlugConverter(SlugConverter):
```

Comment:
Since this isn't really a slug any more, maybe it should just directly subclass [`StringConverter`](https://docs.djangoproject.com/en/2.1/_modules/django/urls/converters/) and be something like `ItemIdConverter`?

Reply:
Done